### ISSUE TITLE
restart crashes i3 when there exists a container with a mark

### ISSUE THREAD

**Issue by infokiller** [i3/i3#2693]
Output of `i3 --moreversion 2>&- || i3 --version`:

```
Binary i3 version:  4.13 (2016-11-08) © 2009 Michael Stapelberg and contributors
Running i3 version: 4.13 (2016-11-08) (pid 27240) abort…)
Loaded i3 config: ~/.i3/config (Last modified: Sun 19 Feb 2017 10:47:34 PM IST, 846466 seconds ago)

The i3 binary you just called: /usr/bin/i3
The i3 binary you are running: i3
```

URL to a logfile as per http://i3wm.org/docs/debugging.html:

http://logs.i3wm.org/logs/5632457692938240.bz2

Note that this is actually an `.xsession-errors` file, since the i3 log doesn't show the crash.
The `.xsession` file simply contains the line `exec i3 --shmlog-size=26214400 -c /etc/i3/config -d all`, so most of the logs are from i3.

**What I did:**

* Open new terminal window
* `i3-msg "mark a"`
* `i3-msg restart`

**What I saw:**

xsession is restarted, i3 seemed to have crashed.

**What I expected instead:**

i3 restarted in-place.

**Comment by Airblader** [comment]
Thanks for your report. This is actually a duplicate of #2511, so closing. :-)

**Comment by infokiller** [comment]
Thanks for the quick response!

The other bug is labelled with 4.12, should that be changed to 4.13?
Also, do you expect the fix to go into the next release?

Thanks!

**Comment by Airblader** [comment]
> The other bug is labelled with 4.12, should that be changed to 4.13?

The tag refers to when it was reported. :-) If it was fixed in 4.13, the ticket would be closed.

> Also, do you expect the fix to go into the next release?

If somebody sends a PR to fix it, sure. :-) Whether I'll have the time to do that myself, I don't know yet.
